libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -114,26 +114,23 @@ private const val MAPBOX_NAVIGATION_TOKEN_EXCEPTION = "A valid token is required
  * todo should we expose a "primaryRouteIndex" field instead of relying on the list's order?
  *
  * @param context activity/fragment's context
- * @param accessToken [Mapbox Access Token](https://docs.mapbox.com/help/glossary/access-token/)
- * @param navigationOptions a set of [NavigationOptions] used to customize various features of the SDK
+ * @param navigationOptions a set of [NavigationOptions] used to customize various features of the SDK.
+ * Use [defaultNavigationOptions] to set default options
  * @param locationEngine used to listen for raw location updates
  * @param locationEngineRequest used to request raw location updates
  */
 class MapboxNavigation
 @JvmOverloads
 constructor(
     private val context: Context,
-    private val accessToken: String?,
-    private val navigationOptions: NavigationOptions = defaultNavigationOptions(
-        context,
-        accessToken
-    ),
```

Comment:
Why did we remove the default `defaultNavigationOptions` argument?

Reply:
default argument for `NavigationOptions` requires `accessToken` that must be under `NavigationOptions` (but not in `MapboxNavigation` constructor)